### ISSUE TITLE
'Add' plugin adds an empty line

### ISSUE THREAD

**Issue by borekb** [dotansimha/graphql-code-generator#4528]
I'm constructing a `schema.graphql` file with the AST plugin and also want to add a comment to the top. This is from my `codegen.yml`:

```yaml
generates:
  schema.graphql:
    schema:
      - schema-source.ts
    plugins:
      - schema-ast
      - add:
          content: '# Generated - DO NOT EDIT'
```

Besides prepending the content, it also appends an empty line which feels like a bug to me:

![Screenshot 2020-08-06 at 10 17 13](https://user-images.githubusercontent.com/101152/89508485-463f2680-d7ce-11ea-9968-a4c29249b13b.png)

**Comment by dotansimha** [comment]
Are you using the v2 of the add plugin? or v1?

**Comment by borekb** [comment]
It's v2, according to my `yarn.lock`.

**Comment by dotansimha** [comment]
Thanks @borekb ! I think I managed to reproduce it and fix it. Pushed to a PR with some other minor bugs: https://github.com/dotansimha/graphql-code-generator/pull/4554

**Comment by dotansimha** [comment]
Fixed in `@graphql-codegen/add@2.0.1`
